src/tools/interop/idt/scripts/bootstrap.sh

```diff
@@ -0,0 +1,19 @@
+#
+#    Copyright (c) 2023 Project CHIP Authors
+#    All rights reserved.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+sudo apt-get update
```

Comment:
It looks like the scripts are not marked executable, so you are required to `source` them? Is that intentional?

Reply:
Aliases are used throughout the scripts and I found it generally easier to manage everything with only one shell.

Reply:
Please reopen if you have any strong opinions here, otherwise will resolve this for now. Everything here can of course change and evolve in the future!